runtimes/adapters/e2b.py

```
@@ -0,0 +1,365 @@
+        try:
+            from agent.loop import AgentRunner
+        except ImportError as exc:  # pragma: no cover - agent loop always present
+            await session.close()
+            raise RuntimeExecutionError(
+                self.RUNTIME_ID,
+                f"AgentRunner import failed: {exc}",
+                spec.task_id,
+            ) from exc
+
+        runner = AgentRunner(
+            workspace_root="/tmp/e2b-host-staging",  # host-side; never written when E2B attached
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/e2b-host-staging"

(S108)